configure.ac

```diff
@@ -178,6 +178,7 @@ ARG_DISBL_SET([pkcs12],         [disable PKCS12 container support plugin.])
 ARG_DISBL_SET([pubkey],         [disable RAW public key support plugin.])
 ARG_DISBL_SET([sshkey],         [disable SSH key decoding plugin.])
 ARG_DISBL_SET([x509],           [disable X509 certificate implementation plugin.])
+ARG_ENABL_SET([openxpki],       [enable OCSP responder accessing OpenXPKI MySQL certificate database.])
```

Comment:
I'd change it to MySQL/MariaDB as well - SQLite is not an option

Reply:
Since they rely on Perl's DBI abstraction, there actually is a [SQLite driver in their repository](https://github.com/openxpki/openxpki/blob/develop/core/server/OpenXPKI/Server/Database/Driver/SQLite.pm) (also several others, that we couldn't don't support but might in the future).